inc/Editor/BlockManagement/BlockRegistration.php

```diff
@@ -83,24 +84,41 @@ public static function register_block_configuration( array $config ): array {
 		$block_path = REMOTE_DATA_BLOCKS__PLUGIN_DIRECTORY . '/build/blocks/remote-data-container';
 
 		// Set available bindings from the display query output mappings.
-		$available_bindings = [];
-		$output_schema = $config['queries'][ ConfigRegistry::DISPLAY_QUERY_KEY ]->get_output_schema();
-		foreach ( $output_schema['type'] ?? [] as $key => $mapping ) {
-			$available_bindings[ $key ] = [
-				'name' => $mapping['name'],
-				'type' => $mapping['type'],
-			];
+		$available_bindings_for_queries = [];
+		$display_queries_to_selectors = $config['display_queries_to_selectors'] ?? [];
+
+		$patterns = $config['patterns'] ?? [];
+
+		// Using array_keys here triggers a psalm error, so it's set to $_ instead.
+		// Supressing the psalm error is a not a good idea, so instead this is the better solution.
+		// ToDo: Fix the psalm error, and see if array_keys could be used here again.
```

Comment:
I haven't been able to solve a psalm error that complains about the types being invalidated and the key being unused, when `array_keys` was used for the loop. So I gave up, and instead just changed the name of the unused selectors to be such that psalm ignores it. If anyone knows this error, feel free to suggest a fix.